Bookish/Controllers/CatalogController.cs

```diff
@@ -13,84 +13,111 @@ public CatalogController(ILogger<CatalogController> logger)
         _logger = logger;
     }
     
-    // GET
-    public IActionResult BookEntry(BookInputModel model)
+    // [HttpPost]

```

Comment:
Shouldn't be commented out, it's valid code and tells the controller that this method is for PUT requests